presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -58,6 +58,12 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: ST_MultiPoint(array(Point)) -> MultiPoint
+
+    Returns a geometry type MultiPoint object formed from an array of points. Empty or null points are ignored. If
```

Comment:
nit: `Returns a MultiPoint geometry object formed from the specified points. If all points in the input array are empty or null, returns null. If there is only one non-empty and non-null point, returns a Point geometry. The returned geometry may not be simple and may contain duplicate points if input array has duplicates.`